ext/POSIX/t/attext.t

```diff
@@ -0,0 +1,70 @@
+use strict;
```

Comment:
This file should be renamed to `atexit.t`.

To make the tests pass, `ext/POSIX/t/atexit.t` should then be added to the `MANIFEST`.

Reply:
I knew that I'm making typos somewhere...